From 57db3ee2c1aa714f6fc88a40043bf531f155843c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Mon, 11 Nov 2013 19:53:59 -0500
Subject: [PATCH] systemd-python: fix booted() and add two functions to docs

For some reason sphinx doesn't want to show inherited C functions.
---
 src/python-systemd/_daemon.c       | 2 +-
 src/python-systemd/docs/daemon.rst | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/python-systemd/_daemon.c b/src/python-systemd/_daemon.c
index 6b84fb8..f0ab16f 100644
--- a/src/python-systemd/_daemon.c
+++ b/src/python-systemd/_daemon.c
@@ -51,7 +51,7 @@ static PyObject* booted(PyObject *self, PyObject *args) {
         assert(args == NULL);
 
         r = sd_booted();
-        if (set_error(r, NULL, NULL))
+        if (set_error(r, NULL, NULL) < 0)
                 return NULL;
 
         return PyBool_FromLong(r);
